docs/guides/building/windows-installer.md

```diff
@@ -0,0 +1,222 @@
+# Windows Installer
+
+Tauri applications for Windows are distributed as Microsoft Installer (`.msi` files). The Tauri CLI bundles your application binary and additional resources in this format if you build on Windows. This guide provides information about available customization options for the installer.
+
+:::info 32-bit Windows
+The Tauri CLI compiles your executable using your machine's architecture by default. Assuming that you're developing on a 64-bit machine, the CLI will produce 64-bit applications. If you need to support 32-bit machines, you can compile your application with a **different** [Rust target][platform support] using the `--target` flag:
+
+<!-- TODO make powershell -->
+
+```console
+tauri build --target i686-pc-windows-msvc
+```
+
+By default Rust only installs toolschains for your machine's target, so you need to install the 32-bit Windows toolchain first: `rustup target add i686-pc-windows-msvc`. You can get a full list of Rust targets by running `rustup target list`.
+:::
+
+## Using a Fixed Version of the Webview2 Runtime
+
+By default, the Tauri installer downloads and installs the Webview2 Runtime if it is not already installed (on Windows 11, WebView2 is preinstalled).
+
+:::note
+You can remove the Webview2 Runtime download check from the installer by setting [`tauri.bundle.windows.wix.skipWebviewInstall`] to `true`.
+
+**Your application WILL NOT work if the user does not have the runtime installed with this option selected.**
+:::
+
+Using a global installation of WebView2 is great for security as Window keeps it updated, but if your end-users have no internet connection or you need a particular version of WebView2, Tauri can bundle the runtime files for you. Keep in mind, this increases the size of Windows installers by **150MB** since your app will include its own copy of Chromium.
+
+1. Download the Webview2 fixed version runtime from the [official website][download the webview2 runtime], a `.cab` file for the selected architecture. In this example, the downloaded filename is `Microsoft.WebView2.FixedVersionRuntime.98.0.1108.50.x64.cab`
+2. Extract the file to the core folder:
+   <!-- Is this powershell? -->
+
+   ```
+   expand .\Microsoft.WebView2.FixedVersionRuntime.98.0.1108.50.x64.cab -F:* ./src-tauri
+   ```
```

Comment:
Is this specific command powershell?

Reply:
should work in cmd.exe too if that's what you mean.

Reply:
yup, that's what I mean